raiden/transfer/log.py

```diff
@@ -83,6 +84,12 @@ def __init__(self, database_path):
             'FOREIGN KEY(statechange_id) REFERENCES state_changes(id)'
             ')'
         )
+        cursor.execute(
+            'CREATE TABLE IF NOT EXISTS state_events ('
+            'identifier integer primary key, source_statechange_id integer NOT NULL, data binary, '
+            'FOREIGN KEY(source_statechange_id) REFERENCES state_changes(id)'
+            ')'
+        )
         self.conn.commit()
 
     def write_transaction(self, data):
```

Comment:
Please, rename this function to be consistent with the other methods, `write_state_change` is more understandable. Also, it would be good to have a batch interface instead of a single entry one.